### PR TITLE
GTT-1088: Fix issue with labels being cut off after adding locale support and deactivate animations

### PR DESCRIPTION
**Pull request by miguelpdiaz8**
## Description

Fix issue with labels being cut off after adding locale support and deactivate animations

## Testing

Tested it in my local environment.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/src/components/ColumnChartWidget.tsx [hunk 1]
```diff
@@ -41,6 +45,20 @@ const ColumnChartWidget = (props: Props) => {
...
+        .yAxisMap;
+      if (yAxisMap && yAxisMap.length) {
+        setYAxisMargin(
+          yAxisMap[0].niceTicks.toLocaleString().match(/,/g)?.length +
+            pixelsByCharacter
+        );
+      }
+    }
+  }, [columnChartRef, columnChartRef.current, chartLoaded]);
+
```
Comment (ferdingler) [inline frontend/src/components/ColumnChartWidget.tsx]: hard to follow what is going on here. Is there any way we can make this easier to read? It also seems to be repeated in the LineChartWidget. Maybe extracting it into a hook would help?

Comment (miguelpdiaz8) [reply frontend/src/components/ColumnChartWidget.tsx]: Added explanation, not extracting it to a hook here.

**Review by ferdingler — APPROVED** [review]
